2020-05-21 16:49:34,243 ERROR: Exception on / [GET] [in c:\users\user\mblog\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\user\mblog\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\user\mblog\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\user\mblog\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\user\mblog\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\user\mblog\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\user\mblog\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\user\mblog\lib\site-packages\flask_login\utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\User\PycharmProjects\mblog\app\routes.py", line 30, in index
    return render_template('index.html', title='Active Gomez',
  File "c:\users\user\mblog\lib\site-packages\flask\templating.py", line 137, in render_template
    return _render(
  File "c:\users\user\mblog\lib\site-packages\flask\templating.py", line 120, in _render
    rv = template.render(context)
  File "c:\users\user\mblog\lib\site-packages\jinja2\environment.py", line 1090, in render
    self.environment.handle_exception()
  File "c:\users\user\mblog\lib\site-packages\jinja2\environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "c:\users\user\mblog\lib\site-packages\jinja2\_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\User\PycharmProjects\mblog\app\templates\index.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "C:\Users\User\PycharmProjects\mblog\app\templates\base.html", line 1, in top-level template code
    {% extends 'bootstrap/base.html' %}
  File "c:\users\user\mblog\lib\site-packages\flask\templating.py", line 60, in get_source
    return self._get_source_fast(environment, template)
  File "c:\users\user\mblog\lib\site-packages\flask\templating.py", line 89, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: bootstrap/base.html
2020-05-21 16:54:12,868 INFO: Flaskblog startup [in C:\Users\User\PycharmProjects\mblog\app\__init__.py:55]
2020-05-22 13:59:24,262 INFO: Flaskblog startup [in C:\Users\User\PycharmProjects\mblog\app\__init__.py:55]
2020-05-22 14:00:18,465 INFO: Flaskblog startup [in C:\Users\User\PycharmProjects\mblog\app\__init__.py:55]
2020-05-22 14:00:48,067 INFO: Flaskblog startup [in C:\Users\User\PycharmProjects\mblog\app\__init__.py:55]
2020-05-22 14:04:00,546 INFO: Flaskblog startup [in C:\Users\User\PycharmProjects\mblog\app\__init__.py:55]
2020-05-22 14:05:43,267 INFO: Flaskblog startup [in C:\Users\User\PycharmProjects\mblog\app\__init__.py:55]
2020-05-22 14:07:58,747 INFO: Flaskblog startup [in C:\Users\User\PycharmProjects\mblog\app\__init__.py:55]
2020-05-22 14:11:33,718 INFO: Flaskblog startup [in C:\Users\User\PycharmProjects\mblog\app\__init__.py:55]
2020-05-22 14:15:23,669 INFO: Flaskblog startup [in C:\Users\User\PycharmProjects\mblog\app\__init__.py:55]
2020-05-22 14:19:29,088 ERROR: Exception on /user/googoosik88 [GET] [in c:\users\user\mblog\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\user\mblog\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\user\mblog\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\user\mblog\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\user\mblog\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\user\mblog\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\user\mblog\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\user\mblog\lib\site-packages\flask_login\utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\User\PycharmProjects\mblog\app\routes.py", line 108, in user
    return render_template('user.html', user=user, posts=posts.items,
  File "c:\users\user\mblog\lib\site-packages\flask\templating.py", line 137, in render_template
    return _render(
  File "c:\users\user\mblog\lib\site-packages\flask\templating.py", line 120, in _render
    rv = template.render(context)
  File "c:\users\user\mblog\lib\site-packages\jinja2\environment.py", line 1090, in render
    self.environment.handle_exception()
  File "c:\users\user\mblog\lib\site-packages\jinja2\environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "c:\users\user\mblog\lib\site-packages\jinja2\_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\User\PycharmProjects\mblog\app\templates\user.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "C:\Users\User\PycharmProjects\mblog\app\templates\base.html", line 1, in top-level template code
    {% extends 'bootstrap/base.html' %}
  File "c:\users\user\mblog\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 1, in top-level template code
    {% block doc -%}
  File "c:\users\user\mblog\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 4, in block "doc"
    {%- block html %}
  File "c:\users\user\mblog\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 20, in block "html"
    {% block body -%}
  File "c:\users\user\mblog\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 23, in block "body"
    {% block content -%}
  File "C:\Users\User\PycharmProjects\mblog\app\templates\base.html", line 50, in block "content"
    {% block app_content %}{% endblock %}
  File "C:\Users\User\PycharmProjects\mblog\app\templates\user.html", line 17, in block "app_content"
    {{ form.hidden_tag() }}
  File "c:\users\user\mblog\lib\site-packages\jinja2\environment.py", line 471, in getattr
    return getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'form' is undefined
2020-05-22 14:22:00,577 INFO: Flaskblog startup [in C:\Users\User\PycharmProjects\mblog\app\__init__.py:55]
2020-05-22 14:22:03,012 ERROR: Exception on /user/googoosik88 [GET] [in c:\users\user\mblog\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\user\mblog\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\user\mblog\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\user\mblog\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\user\mblog\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\user\mblog\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\user\mblog\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\user\mblog\lib\site-packages\flask_login\utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\User\PycharmProjects\mblog\app\routes.py", line 108, in user
    return render_template('user.html', user=user, posts=posts.items,
  File "c:\users\user\mblog\lib\site-packages\flask\templating.py", line 137, in render_template
    return _render(
  File "c:\users\user\mblog\lib\site-packages\flask\templating.py", line 120, in _render
    rv = template.render(context)
  File "c:\users\user\mblog\lib\site-packages\jinja2\environment.py", line 1090, in render
    self.environment.handle_exception()
  File "c:\users\user\mblog\lib\site-packages\jinja2\environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "c:\users\user\mblog\lib\site-packages\jinja2\_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\User\PycharmProjects\mblog\app\templates\user.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "C:\Users\User\PycharmProjects\mblog\app\templates\base.html", line 1, in top-level template code
    {% extends 'bootstrap/base.html' %}
  File "c:\users\user\mblog\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 1, in top-level template code
    {% block doc -%}
  File "c:\users\user\mblog\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 4, in block "doc"
    {%- block html %}
  File "c:\users\user\mblog\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 20, in block "html"
    {% block body -%}
  File "c:\users\user\mblog\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 23, in block "body"
    {% block content -%}
  File "C:\Users\User\PycharmProjects\mblog\app\templates\base.html", line 50, in block "content"
    {% block app_content %}{% endblock %}
  File "C:\Users\User\PycharmProjects\mblog\app\templates\user.html", line 17, in block "app_content"
    {{ form.hidden_tag() }}
  File "c:\users\user\mblog\lib\site-packages\jinja2\environment.py", line 471, in getattr
    return getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'form' is undefined
